src/components/articleTeaser/index.tsx

```diff
@@ -27,10 +28,10 @@ const ArticleTeaser: FC<Props> = ({
       <a href={url} target="_blank" rel="noopener noreferrer">
         <Stack direction="column" spacing="lg" align="center">
           <chakra.img maxW={maxImgW} src={imageUrl} />
-          <Stack direction="column" spacing="sm">
+          <Box as={Stack} direction="column" spacing="sm" width="full">
```

Comment:
I am a bit uneasy about this.
The way `as` works is that `Stack` component will be rendered instead of `Box` and all props will be passed to the `Stack`. So the solution only works because despite `Pick`ing from the chakra props we forward everything to the chakra component. The moment we change that logic this would break. I'd recommend either:
- using chakra stack directly
- whitelisting the `width` prop on our component
- using `Flex` component or even `display` properties of `Box`

Reply:
I think it makes sense to whitelist width. It is a UI component so you should be able to use UI props